backend/src/auth/RIGHTS.ts

```diff
@@ -38,6 +38,7 @@ export enum RIGHTS {
   OPEN_CREATIONS = 'OPEN_CREATIONS',
   USER = 'USER',
   GMS_USER_PLAYGROUND = 'GMS_USER_PLAYGROUND',
+  CIRCLES_AUTO_LOGIN = 'CIRCLES_AUTO_LOGIN',
```

Comment:
Wäre hier nicht besser den Dienst (HUMHUB) statt ein fachliches Teilfeature wie Kreise (CIRCLES) von HumHub in der Konstante zu verwenden?